### DISCUSSION TITLE
arm (gnueabihf and musleabihf) binaries

### DISCUSSION THREAD

**Discussion by yelkarama** [BurntSushi/ripgrep#2698] [General] [answered]
Hi
Can you please produce 32bit arm (gnueabihf and musleabihf) binaries? 
Thanks

**Comment by BurntSushi** [comment, marked as answer]
If you or someone else can make a PR and do the testing required to demonstrate that the release workflow works, then I can probably get on board with that.

**Reply by yelkarama** [reply]
Got this error:
[https://pastebin.com/RmNw0rqp](https://pastebin.com/RmNw0rqp)
It's related to pcre2. No error when I remove the `--features pcre2` switch 
However, I'm able to build [pcre2 ](https://github.com/BurntSushi/rust-pcre2.git) itself without issues:
[https://pastebin.com/zy6D1PcJ](https://pastebin.com/zy6D1PcJ)

**Reply by yelkarama** [reply]
Just found out you included pcre2 lib in this repo (./crates/pcre2). I'm able to successfully build that too. 
However, rebuilding ripgrep still yield the same error.

**Reply by BurntSushi** [reply]
That looks like a JIT error. You probably need to submit a PR to the `pcre2-sys` crate that disables the JIT for your targets: https://github.com/BurntSushi/rust-pcre2/blob/8b6f7a6d69efab14269218f36da7b145eb1f2708/pcre2-sys/build.rs#L99-L121

**Reply by yelkarama** [reply]
Is there a way I can test locally to confirm before submitting the PR?

**Reply by BurntSushi** [reply]
Add this to ripgrep's `Cargo.toml`:

```toml
[patch.crates-io]
pcre2-sys = { path = "/home/yelkarama/path/to/fork/of/rust-pcre2/pcre2-sys" }
```

And that should do it.

**Reply by yelkarama** [reply]
[PR](https://github.com/BurntSushi/rust-pcre2/pull/37) created

**Reply by BurntSushi** [reply]
OK, merged, released and I've pushed an update to ripgrep's `Cargo.lock` to use `pcre2-sys 0.2.8`.

**Reply by BurntSushi** [reply]
And this is now merged via https://github.com/BurntSushi/ripgrep/pull/2702.
